gemfiles/rails_7.gemfile

```diff
@@ -0,0 +1,6 @@
+source "https://rubygems.org"
+
+gem "rails", github: "rails/rails", branch: "7-0-stable"
```

Comment:
Rails 7.0.0 is not compatible with Ruby 3.1.0, so need to use Rails `7-0-stable` branch until Rails 7.0.1 is released.